src/features/AdminPage/TeamPage/TeamModal/TeamModal.component.tsx

```
@@ -32,7 +31,14 @@ import TeamLink from '@/features/AdminPage/TeamPage/TeamLink.component';
 import Audio from '@/models/media/audio.model';
 import Image from '@/models/media/image.model';
 
-import POPOVER_CONTENT from '../../JobsPage/JobsModal/constants/popoverContent';
+import { UploadChangeParam } from 'antd/es/upload';
+import combinedImageValidator, { checkFile } from '@components/modals/validators/combinedImageValidator';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'combinedImageValidator'. Consider to delete it or rename it.

'combinedImageValidator' is defined here:


(lint/suspicious/noRedeclare)